plugin-server/src/utils/db/db.ts

```diff
@@ -298,7 +298,7 @@ export class DB {
                 const value = await tryTwice(
                     async () => await client.get(key),
                     `Waited 5 sec to get redis key: ${key}, retrying once!`
-                )
+                ) // TODO: add a test getting 'undefined' here for groups making sure we have the props properly filled out, local testing showed we don't
```

Comment:
& null too

Reply:
local testing: if we got `undefined` here (which I assume can happen if the get call failed in some way  🤷‍♀️  ) then we'd return 'missing', i.e. the defaultValue, which we then would interpret as we had an entry `null` in the cache (which is what we were writing when we didn't have any props info for the group to avoid hammering postgres in those cases)